src/main/scala/nl.knaw.dans.easy.dd2d/InboxProcessor.scala

```diff
@@ -26,7 +26,7 @@ class InboxProcessor(inbox: Inbox) extends DebugEnhancedLogging {
     trace(())
     val ingestTasks = new PassiveTaskQueue[Deposit]()
     logger.info("Enqueuing deposits found in inbox...")
-    inbox.enqueue(ingestTasks, Some(new DepositSorter))
+    inbox.enqueue(ingestTasks, Some(new DepositSorter)).get // TODO: better error handling
```

Comment:
unsafeGetOrThrow from the dans-scala-lib

Reply:
From the Scaladocs of `unsaveGetOrThrow`:

> Note that this behavior is the same as `Try.get`, but that this explicitly acknowledges the
> potential danger of throwing an `Exception`.

I guess it is slightly better to acknowledge that what you are doing is wrong...